semver-from-git.rkt

```diff
@@ -2,8 +2,25 @@
 
 (require racket/system)
 
+(define semver-logger
+  (make-logger 'semver
+	       (current-logger)
+	       'debug
+	       'semver))
+
+(define semver-log-receiver
+  (make-log-receiver semver-logger
+		     'debug
+		     'semver))
+
 (define (debug tag v msg)
-  (displayln (~a tag "=>" v " --- " msg)))
+  (when (verbose?)
+    (log-message semver-logger
+		 'debug
+		 'semver
+		 (~a tag "=>" v " --- " msg))
+    (let [(v (sync semver-log-receiver))]
```

Comment:
This may need to be abstracted if you ever add a `warning` or other level specific emitter function.